CHANGELOG.md

```diff
@@ -1,3 +1,34 @@
+## **5.65.0** (2021-11-26)
```

Comment:
This is a technical release, please change the PR title and this line:

```suggestion
## **next**
```

This is necessary because we'll probably do a second technical release today.